sdks/go/test/build.gradle

```diff
@@ -166,3 +174,33 @@ task ulrValidatesRunner {
     }
   }
 }
+
+// A method for configuring a cross-language validates runner test task,
+// intended to be used in calls to createCrossLanguageValidatesRunnerTask.
+ext.goIoValidatesRunnerTask = { proj, name, scriptOpts ->
+  return proj.tasks.create(name: name) {
+    group = "Verification"
+
+    dependsOn ":sdks:go:test:goBuild"
+    dependsOn ":sdks:go:container:docker"
+    dependsOn ":sdks:java:container:java11:docker"
```

Comment:
This is not necessary. `setupTask` depends on Java container build and `goIoValidatesRunnerTask` depends on `setupTask`.

Reply:
Done.